lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2022,6 +2033,190 @@ def test_get_students_features_cohorted(self, is_cohorted):
 
         self.assertEqual('cohort' in res_json['feature_names'], is_cohorted)
 
+    def test_enrollment_report_features_csv(self):
+        """
+        test to generate enrollment report.
+        enroll users, admin staff using registration codes.
+        """
+        invoice_transaction = InvoiceTransaction(
```

Comment:
Call `InvoiceTransaction.objects.create()` and you won't need the call to `save()`.